src/marten/db/query/sql/query.cr

```diff
@@ -206,6 +206,59 @@ module Marten
             !(@limit.nil? && @offset.nil?)
           end
 
+          def sum(raw_field : String)
+            column_name = if !raw_field.nil?
+                            field_path = verify_field(raw_field.to_s)
+                            relation_field_path = field_path.select { |field, _r| field.relation? }
+
+                            if relation_field_path.empty? || field_path.size == 1
+                              "#{Model.db_table}.#{field_path.first[0].db_column!}"
+                            else
+                              join = ensure_join_for_field_path(relation_field_path)
+                              join.not_nil!.column_name(field_path.last[0].db_column!)
+                            end
+                          end
```

Comment:
Could we simplify these lines since `raw_field` will always be a `String`?

For example:

```suggestion
            field_path = verify_field(raw_field.to_s)
            relation_field_path = field_path.select { |field, _r| field.relation?
            
            column_name = if relation_field_path.empty? || field_path.size == 1
              "#{Model.db_table}.#{field_path.first[0].db_column!}"
            else
              join = ensure_join_for_field_path(relation_field_path)
              join.not_nil!.column_name(field_path.last[0].db_column!)
            end
```